widget/table.go

```diff
@@ -67,6 +89,50 @@ func (t *Table) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
+// SetSelected will mark the specified cell (at row, col) to be marked as selected.
+func (t *Table) SetSelected(row, col int) {
+	t.SelectedRow = row
+	t.SelectedColumn = col
+
+	t.scrollToVisible(row, col)
+	if t.OnCellSelected != nil {
+		t.OnCellSelected(row, col)
+	}
+
```

Comment:
I'm not sure we should be calling OnCellSelected when we are quite literally doing the opposite of what the api suggests by unselecting?

Reply:
I think you might have commented on the wrong method?

Reply:
yes sorry